utils/get_test_info.py

```diff
@@ -0,0 +1,118 @@
+# coding=utf-8
```

Comment:
This new file contains several methods which will be helpful, as mentioned in the PR description.